### DISCUSSION TITLE
apfs-dmg : windows fuse dokan/winfsp/libfsapfs

### DISCUSSION THREAD

**Discussion by P5-2005** [blacktop/ipsw#345] [Ideas]
wonder if possible to add dmg apfs mount part on windows?!
for fuse part on windows there is :
dokan
winfsp

libfsapfs(can mount dmg apfs, like : ramdisk if libfuse exist), still not support mount only cryptex apfs dmg thing

**Comment by blacktop** [comment]
I'd be interested in adding support.  I personally have 0 access to windows machines/VMs currently so it's the hardest platform for me to test on (but I guess I could use the Github Actions CI)

**Comment by blacktop** [comment]
I do have a pure Go APFS parser here - https://github.com/blacktop/go-apfs that should work on Windows as well, but it broke when the NEW lzbitmap compression was added and I don't believe I got around to finishing it. However, it is definitely on my roadmap to finish and have full APFS access to all the OSes etc.
